tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -136,6 +136,7 @@ class DuckDBPyConnection:
     def load_extension(self, extension: str) -> None: ...
     def pl(self, chunk_size: int = ..., connection: DuckDBPyConnection = ...) -> polars.DataFrame: ...
     def torch(self, connection: DuckDBPyConnection = ...) -> dict: ...
+    def tf(self, connection: DuckDBPyConnection = ...) -> dict: ...
```

Comment:
Is tf a well known shortening of tensor flow? My first thought at seeing it is "transform"

Reply:
Im no ML guy but from their tutorials it seems like a common alias 😅

Reply:
I can imagine in the ecosystem that uses
`import pandas as pd`
`import numpy as np`
that `import tensorflow as tf` is a commonly used acronym